elementary/monitor/dbt_project/macros/get_test_results.sql

```diff
@@ -1,4 +1,4 @@
-{%- macro get_test_results(days_back = 7, invocations_per_test = 720, disable_passed_test_metrics = false) -%}
+{% macro prepare_test_results(days_back = 7, invocations_per_test = 720, disable_passed_test_metrics = false) %}
```

Comment:
change so 
- `prepare_test_results` - contains the `adapter.dispatch` call instead of containing the default behavior
- `default__prepare_test_results` - contains the default behavior
- `get_test_results` - calls `prepare_test_results` instead of doing `adapter.dispatch`